lib/utils/logger.ts

```diff
@@ -132,6 +134,12 @@ export class Logger {
 	// Generic output method for either type.
 	private output(level: number, classLevel: number, levelStrings: string[], message: string): void {
 		if (level <= classLevel) {
+			if (!_.isEmpty(this._secrets) && process.env.PROCBOT_LOG_SECRETS !== 'true') {
```

Comment:
Were we not going to base this on the output level? I don't really like having a new envvar specifically for logging secrets.